build/pipelines/templates/code-mirror.yml

```diff
@@ -0,0 +1,21 @@
+trigger:
+  branches:
+    include:
+    # Keep this set limited as appropriate (don't mirror individual user branches).
+    - main
+    - release/v1.x
```

Comment:
@fabiocav did you add rule set to this repo to protect release branches? If so, we do have the option of using `release/*` here.

Reply:
But can always make that glob change later.

Reply:
The change is in place. So we can update the pattern.